servers/services/pom.xml

```diff
@@ -60,5 +60,11 @@
       <groupId>jakarta.enterprise</groupId>
       <artifactId>jakarta.enterprise.cdi-api</artifactId>
     </dependency>
+      <dependency>
```

Comment:
I'm not sure why this dependency is being added as I don't see a code change using it, `datanucleus` which is something for DB persistence should not be required

Reply:
IDE just being dumb and helpfully adding deps for me. Will remove it.